js/script.js

```diff
@@ -1,15 +1,215 @@
-// Javascript Code.
-$(document).ready(function () {
-  // $.get("_posts/2014-11-09.json", function(data){
-  //
-  //   var posts = JSON.parse(data);
-  //   $(".posts").html(posts['posts'][0]['title']);
-  //
-  // });
-
-  $(this).load("_posts/posts.json", function(data){
-    var posts = JSON.parse(data);
-    $(".posts").html(posts['posts'][0]['title']);
-    //
-  });
-});
+// Javascript Code.
+
+//This is the search box on the page
+$(document).ready(function(){
+    $('.buttonsearch').click(function() {
+        var toAdd = $('input[name=checkListItem]').val();   
+        $('.list').append('<div class="item">' + toAdd + '</div>')
+    });
+});
+
+//This is the search box on the page
+
+
+

```

Comment:
Good work with your page so far! There are some improvements that will help you out in the future. For example, indenting your code makes it much more readable, you can read more about it and other good coding practices at: http://en.wikipedia.org/wiki/Programming_style.
This is a very important habit to have because it makes your code much more maintainable.